### DISCUSSION TITLE
How to use Tailpress with Local

### DISCUSSION THREAD

**Discussion by wimdeblauwe** [tailpress/tailpress#208] [Q&A]
I generated a Tailpress theme using the installer. I run my website using https://localwp.com/. I enabled browsersync by uncommenting the lines at https://github.com/jeffreyvr/tailpress/blob/master/webpack.mix.js#L21:L26

When I run `npm run watch`, my browser opens with my website. But I don't see the theme in the list of themes. How do I add the theme in such a way that the live reload will work?

**Comment by wimdeblauwe** [comment]
I don't know if it is the intention, but I just added a symbolic link to the themes folder of the wordpress installation. That way, the theme is updated automatically. I do have to manually refresh the browser, that is annoying. I suppose it should normally refresh automatically?

**Comment by ReyJust** [comment]
I am also using Local as dev environment. I simply run the installer in the theme directory `~/Local Sites/website_name/app/public/wp-content/themes/`, and everything was already working out of the box.
I do notice however that there was hot reloading, but I am unsure if it's suposed to be or not.

**Comment by luki-zz** [comment]
Hello, 
I'm using Local to. Uncommented the lines in the webpack.mix.js and chenged urls to mine.
When started npm run watch the site open in browser, but the hot reload is not working. Do You know where to search for solutions to this problem?
